### PR TITLE
Fixed Broken Link on the Template Documentation Web Page

### PR DESCRIPTION
**Pull request by a4aditya**
Fixed the broken link by adding the new link to all of the web pages.

# Issue #324 
<Cicero Documentation link is broken on every Template Documentation Web page >

### Changes
- Changed the broken link to the new links as suggested by @irmerk 

Please Review @irmerk

### CODE REVIEW

**Comment by a4aditya** [comment]
Thanks for the review Jerome! Will surely change the text. Can u suggest something that should be replaced by the original text.

**Comment by jeromesimeon** [comment]
> Thanks for the review Jerome! Will surely change the text. Can u suggest something that should be replaced by the original text.

I'm happy to let you suggest something that makes sense to you.

**Comment by jeromesimeon** [comment]
@a4aditya Any news on this PR?

**Comment by a4aditya** [comment]
@jeromesimeon Jerome I wasn't able to find proper resource for that.. could you suggest something to me that is mindful enough.

**Comment by jeromesimeon** [comment]
hi @a4aditya how about "Please refer to the Accord Project documentation for how to <a href="https://docs.accordproject.org/docs/started-hello">use a template</a> or <a href="hhttps://docs.accordproject.org/docs/tutorial-nodejs.html">integrating within your application</a>."

**Comment by jeromesimeon** [comment]
> The text according to me can't be changed because of the heading that precedes the text. So I believe changing the Link in the template is more thoughtful.
> 

Sorry I didn't understand this, can you explain what ou mean "the text can't be changed" ?

**Comment by a4aditya** [comment]
What I mean by 'the text can't be changed' is that changing the text doesn't make a sense because of the Heading that preceeds the text.

**Comment by a4aditya** [comment]
What I feel is giving a link to the Cicero API would be helpful

how about 'Please refer to the Accord Project documentation for how to [use a template](https://docs.accordproject.org/docs/started-hello) or [integrating within your application](https://docs.accordproject.org/docs/cicero-api.html)

**Comment by jeromesimeon** [comment]
> What I mean by 'the text can't be changed' is that changing the text doesn't make a sense because of the Heading that preceeds the text.

Nothing prevents us to change both the title and text.

I think users will need to review the documentation anyway to use the API. The tutorial for how to use a template with node.js sounds like the right level to me.

**Comment by a4aditya** [comment]
I think that's a valid point, keeping that in mind changing the link to the node.js tutorial would be great 
Should I edit the suggested changes i.e.
'Please refer to the Accord Project documentation for how to [use a template](https://docs.accordproject.org/docs/started-hello) or [integrating within your application](https://docs.accordproject.org/docs/tutorial-nodejs.html)

**Comment by jeromesimeon** [comment]
> I think that's a valid point, keeping that in mind changing the link to the node.js tutorial would be great
> Should I edit the suggested changes i.e.
> 'Please refer to the Accord Project documentation for how to [use a template](https://docs.accordproject.org/docs/started-hello) or [integrating within your application](https://docs.accordproject.org/docs/tutorial-nodejs.html)

sure!

**Comment by jeromesimeon** [comment]
please remember you should not edit all the HTML files by hands. I think the only place that needs a fix is the template.

**Comment by a4aditya** [comment]
for sure indeed!!

**Comment by jeromesimeon** [comment]
@a4aditya Any update on this?

**Comment by a4aditya** [comment]
I apologies for the delay, I have changed the Developers.njk file to the suggested changes.

File: views/developers.njk [hunk 1]
```diff
@@ -81,8 +81,7 @@
     <h2 class="title is-2">Technical Integration</h2>
     <div class="content">
         <p>
-            Please refer to the <a href="https://docs.accordproject.org/docs/cicero.html">Cicero documentation</a> 
-            for details and examples of how to integrate a call to a Cicero template into your application.
+            Please refer to the Acccord Project documentation for how to <a href="https://docs.accordproject.org/docs/started-hello">use a template</a> or <a href="https://docs.accordproject.org/docs/tutorial-nodejs.html">integrating within your application</a>.
```
Comment (jeromesimeon) [inline views/developers.njk:84]: This reads odd. Can you please use `integrate it` rather than `integrating`.

**Pull request closed without merging**